src/plugins/navigation-data/public/webcomponents/attitude-indicator.html

```diff
@@ -0,0 +1,632 @@
+        <text
+          xml:space="preserve"
+          style="font-style:normal;font-weight:normal;font-size:13.40397072px;line-height:100%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;baseline-shift:baseline;display:inline;fill:#ffffff;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+          x="645.66846"
+          y="980.81873"
+          id="text12240-0"
+          class="tick-text"
+          sodipodi:linespacing="100%"
+          transform="scale(0.96506539,1.0361992)"><tspan
+            sodipodi:role="line"
+            id="tspan12242-8"
+            x="645.66846"
+            y="980.81873"
+            style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;line-height:100%;font-family:sans-serif;-inkscape-font-specification:'sans-serif Bold';text-align:center;writing-mode:lr-tb;baseline-shift:baseline;text-anchor:middle;fill:#ffffff;fill-opacity:1;stroke:none;stroke-opacity:1">-90°</tspan></text>
+      </g>
+      </g>
+      <g id="crosshair"
+        inkscape:label="Layer 1"
+        inkscape:groupmode="layer"
+        id="layer1"
+        transform="translate(-92.52822,-300.87297)">
+        <circle
+          style="display:inline;opacity:1;fill:#b71c1c;fill-opacity:1;stroke:#ffffff;stroke-width:1.9704386;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1"
+          id="path8517"
+          
+          cx="187.11037"
+          cy="306.52222"
+          r="5.0147805"
+          transform="matrix(0.99999824,0.00187749,-0.00187749,0.99999824,0,0)" />
+        <rect
+          style="opacity:1;fill:#ffffff;fill-opacity:1;fill-rule:evenodd;stroke:#ffffff;stroke-width:0.34768012;stroke-linejoin:round;stroke-opacity:1"
+          id="rect11102-5"
+          width="29.152319"
+          height="1.7523198"
+          x="152.51102"
+          y="305.99683" />
+        <rect
+          style="opacity:1;fill:#ffffff;fill-opacity:1;fill-rule:evenodd;stroke:#ffffff;stroke-width:0.34768012;stroke-linejoin:round;stroke-opacity:1"
+          id="rect11102-5-8"
+          width="29.152319"
+          height="1.7523198"
+          x="192.03831"
+          y="305.99683" />
+        </g>
+      </svg>
+
+      <div id="pitch-slider">
+        <paper-slider id="hudPosSlider" min="0" max="100" pin value="0"></paper-slider>
+      </div>
+  </template>
+
+  <script>
+      var map = function (value, in_min, in_max, out_min, out_max) {
+        return (value - in_min) * (out_max - out_min) / (in_max - in_min) + out_min;
+       };
+
+       var isDemo = false;
+       var demoCounter = 0;
+
+      (function () {    
+      Polymer({
+        is: 'attitude-indicator',
+        properties: {
+          pitchValue: {
+            type: Number,
+            value: 0
```

Comment:
Be sure to set the default values to a function that returns the default instead of an actual value to make it individual for each instance of the control